src/boot/efi/stub.c

```diff
@@ -277,6 +477,31 @@ static EFI_STATUS run(EFI_HANDLE image) {
                 mangle_stub_cmdline(cmdline);
         }
 
+        /* If we have any extra command line to add via PE addons, load them now and append, and
+         * measure the additions separately, after the embedded options, but before the smbios ones,
+         * so that the order is reversed from "most hardcoded" to "most dynamic". The global addons are
+         * loaded first, and the image-specific ones later, for the same reason. */
+        err = cmdline_append_and_measure_addons(image,
```

Comment:
usually we break before the first param, not after, except if the function name is so short that the parameter would be visually disconnected from the function call then